### ISSUE TITLE
question about capsule tenant owner default Role

### ISSUE THREAD

**Issue by TinySong** [clastix/capsule#573]
# Bug description

I have some questions about the default role of tenant owner as described in https://capsule.clastix.io/docs/general/tutorial#assign-tenant-ownership. Maybe this is not a bug.

as the description above. when creating the tenant using the yaml
```
apiVersion: capsule.clastix.io/v1beta1
kind: Tenant
metadata:
  annotations:
    clusterrolenames.capsule.clastix.io/user.alice: editor,manager
  name: oil
spec:
  owners:
    - kind: User
      name: alice
```
question 1:
alice the tenant owner will assign  `ClusterRole/editor` and `ClusterRole/manager`. In other word,  the clusterRole `editor` 
`manager` should I create manually? 

question 2:
<img width="778" alt="image" src="https://user-images.githubusercontent.com/16032654/170829058-feab2b09-44c6-4c0e-bdc0-881c053dd5c5.png">
as described in the snapshot  Captured from  https://capsule.clastix.io/docs/general/tutorial#assign-tenant-ownership, 
`kubectl get rolebindings.rbac.authorization.k8s.io` rolebinding is namespace scope. the rolebinding should be created after a namespace created by aclice.

the documentation makes me confusing.

**Comment by TinySong** [comment]
cc @prometherion  need some help!

**Comment by prometherion** [comment]
> question 1:
> alice the tenant owner will assign ClusterRole/editor and ClusterRole/manager. In other word, the clusterRole editor
manager should I create manually?

tl;dr; yes

This feature has been designed to address a fine-grained multi-tenancy implementation. Let's imagine the case where you're working in a company that must address specific security guidelines and your security department states that the `admin` role is too much wide for their standards. They're requiring you to implement a new one, named `limited_admin` that has limitations and meets their criteria. To address this, the Tenant Owners must be bound to that specific ClusterRole that you have created in advance, since also the `admin` one, by default, is installed by Kubernetes itself.

> question 2:
> as described in the snapshot Captured from https://capsule.clastix.io/docs/general/tutorial#assign-tenant-ownership,
`kubectl get rolebindings.rbac.authorization.k8s.io rolebinding` is namespace scope. the rolebinding should be created after a namespace created by aclice.

You're right, we missed to specify that those `RoleBinding` resources are living in a Namespace owned by a Tenant, we should address it to the documentation by adding the `--namespace` or `-n` flag to the command.

Would you like to contribute @TinySong?

**Comment by TinySong** [comment]
@prometherion  PR has submitted, please see if there is anything that needs to be added. 😄
